Installer/MainInstaller.sh

```diff
@@ -92,10 +92,4 @@ cd ../..
 ./configure
 make -j4
 make install -j4
-cd ..
-
-#install wiringPi
-git clone https://github.com/WiringPi/WiringPi.git
-cd WiringPi
-git pull origin
-./build
+cd ..
```

Comment:
Missing trailing empty line. Also this file would be better written if `cd` wasn't ever used as a whole script change but scoped like
```bash
(cd WiringPi &&
   git pull origin &&
   ./build)
```